Tasks/AzureRmWebAppDeploymentV4/operations/TaskParameters.ts

```diff
@@ -119,6 +119,7 @@ export class TaskParametersUtility {
         taskParameters.PublishProfilePath = tl.getInput('PublishProfilePath', true);
         taskParameters.PublishProfilePassword = tl.getInput('PublishProfilePassword', true);
         taskParameters.Package = new Package(tl.getPathInput('Package', true));
+        taskParameters.SetParametersFile = tl.getPathInput('SetParametersFile', false);
```

Comment:
SetParametersFile var name sounds like a method, can be renamed to something like ParametersFile/ParametersFilePath/ParamsFile